tests/integration/conftest.py

```diff
@@ -79,9 +79,12 @@ def convert2rhel(shell):
     >>>     c2r.expect("Kernel is compatible with RHEL")
     >>> assert c2r.exitstatus == 0
 
+    Unregister set to false means, that system after conversion woudn't be unregistered
+    from subscription-manager. Then that is needed to be done manually.
     """
 
     @contextmanager
+    # def factory(options: str, timeout: int = 60 * 60, unregister: bool = True) -> ContextManager[pexpect.spawn]:
```

Comment:
There I have a problem. I need to skip `subscription-manager unregister` which is ran directly after conversion. My plan was to create optional argument `unregister` by default set to True. In this case I would set it to False and unregister the system manually after the checks.

But I don't know how to write that. You can see some attempts in comments, but always I got an error.